build/Tests/UnitTests/CoreGraphics.Drawing/CoreGraphics.Drawing.UnitTests.vcxproj

```diff
@@ -240,6 +240,7 @@
     </ClangCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics.drawing\CGTextDrawing.cpp" />
```

Comment:
>..\..\..\..\t [](start = 27, length = 13)

for neatness sake, move it to the grouping with the other test files or create grouping for CGText*
Given this is pretty much Enter point and frameworks.